👷 [ci]: Ready for Release (beta) (#594)
withstudiocms/studiocms: packages/studiocms/CHANGELOG.md

- [#589](https://github.com/withstudiocms/studiocms/pull/589) [`688dd08`](https://github.com/withstudiocms/studiocms/commit/688dd08b1062b47132d90d39c6a5a946f40bb8b6) Thanks [@Adammatthiesen](https://github.com/Adammatthiesen)! - Refactor lib/auth to Utilize Effect-TS

- [#601](https://github.com/withstudiocms/studiocms/pull/601) [`f13e396`](https://github.com/withstudiocms/studiocms/commit/f13e396308426df2b8f08f1502f3c43c20c7241e) Thanks [@Adammatthiesen](https://github.com/Adammatthiesen)! - Refactor middleware, add new effect utils, and effect cleanup

  - Middleware files are now Effects.
  - Updated all current effects to Effect Services, and created new effect utils in `lib/effects/`
    - New SMTP Effect for wrapping `nodemailer`
    - New Logger utils for Effect
    - Updated lib/auth effects
    - Updated lib/mailer effect
    - Updated lib/notifier effect

- [#614](https://github.com/withstudiocms/studiocms/pull/614) [`88e624d`](https://github.com/withstudiocms/studiocms/commit/88e624df79ac3d2a11c9bbe29d31495ab7ef9a7e) Thanks [@Adammatthiesen](https://github.com/Adammatthiesen)! - Refactor code and config structure.

  #### BREAKING CHANGES

>   New Config structure. please verify new configs when updating. configs can also be converted from `studiocms.config.js` to `studiocms.config.ts` and they will now work while being fully typed!

Reviewer: ```suggestion
StudioCMS has a new configuration structure. Please verify your configuration after updating. Configurations can now also be converted from `studiocms.config.js` to `studiocms.config.ts` and they will work while being fully typed!
```

Reviewer 2: fixed in https://github.com/withstudiocms/studiocms/pull/627
